augur/types.py

```diff
@@ -14,21 +33,22 @@ class DataErrorMethod(enum.Enum):
 
 
 @enum.unique
-class ValidationMode(enum.Enum):
+class EmptyResultsReportingMethod(ArgparseEnum):
     """
-    Enum representation of string values that represent how validation should
-    be handled.
+    Enum representation of string values that represent how empty results should
+    be reported.
     """
     ERROR = 'error'
     WARN  = 'warn'
     SKIP  = 'skip'
```

Comment:
This is a minor point, but "skip" doesn't quite match the context in augur filter like it did in the augur validation context. I'd think "ignore" might be a more appropriate verb for the empty filter results.

Reply:
I agree that "skip" matches "reporting", but I'm suggesting that "reporting" reflects the internals of the code and not what the user is asking augur filter to do.

As a user, I want to tell augur filter what to do (or how to handle the case) when there are no results in the output: throw an error, print a warning, or do nothing (i.e., ignore the empty output). Asking augur filter to "skip reporting empty results" doesn't quite match how I normally think about responding to errors and warnings (which would be to "ignore empty results"). There might be a better verb than "ignore" or "skip" that captures "do nothing" in a single word though...

By the same logic, asking augur export to "skip validation" matches what I want as a user (when I get the annoying augur version mismatch errors). I don't think I would ask augur export to "ignore validation" because I would want to avoid running validation altogether.

Reply:
Huh, I'm thinking the opposite, where "handled" reflects internals of the code.

As a user, I'm telling augur filter how to report empty results to me. Report it as an error, a warning, or skip the report altogether.

Reply:
Ah, well, your examples of what makes sense to you seem inconsistent to me... :upside_down_face: so I think we either have different connotations here or else we're accidentally talking past each other.

This is minor, and I'm happy to drop it in favor of whatever @joverlee521 and you think is best.

Reply:
> There might be a better verb than "ignore" or "skip" that captures "do nothing" in a single word though...

We can continue the augur curate model and go with `silent` 😄

Reply:
Heh, yeah, `silent` could work, too. Maybe it's too minor to dwell on more. I'm happy with whatever you prefer, @joverlee521.

Reply:
Updated to `silent` in the forced-push.